apps/studio/electron/main/assets/fonts/layout.ts

```
@@ -293,11 +294,16 @@ export async function updateFontInLayout(
     filePath: string,
     font: Font,
     targetElements: string[],
-): Promise<void> {
+): Promise<null | CodeDiff> {
     let updatedAst = false;
     const fontClassName = `font-${font.id}`;
     let result = null;
 
+    const content = await readFile(filePath);
+    if (!content) {
+        return null;
```

Should warn when file not found or better yet return a failure instead of failing silently